api-server-rest/build.rs

```diff
@@ -101,6 +118,17 @@ fn main() -> std::io::Result<()> {
         .run()
         .expect("Generate ttrpc protocol code failed.");
 
+    // Fix clippy warnings of code generated from ttrpc_codegen
+    replace_text_in_file(
+        "src/ttrpc_proto/attestation_agent_ttrpc.rs",
+        "#![allow(box_pointers)]\n",
```

Comment:
btw I have been fixing this upstream https://github.com/containerd/ttrpc-rust/pull/235. But seem it would take a long time so we need this PR.